src/com/lftechnology/java/training/sanish/generics/GenericTest.java

```diff
@@ -0,0 +1,142 @@
+package com.lftechnology.java.training.sanish.generics;
+
+import com.lftechnology.java.training.niraj.fibonacci.Constant;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'com.lftechnology.java.training.niraj.fibonacci.Constant'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AUselessImportCheck)